packages/database/src/migrations/20210512001037-AddPSSSAccumulatedConfirmedCasesPerSyndromeIndicators-modifies-data.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+import { generateId, insertObject, arrayToDbString } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+const SYNDROMES = ['AFR', 'DIA', 'ILI', 'PF', 'DLI'];
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const getIndicatorCode = syndrome => `PSSS_Accumulated_${syndrome}_Confirmed_Cases`;
+
+exports.up = async function (db) {
+  for (const syndrome of SYNDROMES) {
+    const indicator = {
+      id: generateId(),
+      code: getIndicatorCode(syndrome),
+      builder: 'analyticArithmetic',
+      config: {
+        formula: `PSSS_Confirmed_${syndrome}_Cases`,
+        aggregation: ['FINAL_EACH_WEEK', 'SUM_PER_ORG_GROUP'],
+      },
+    };
+
+    await insertObject(db, 'indicator', indicator);
+  }
+
+  return null;
+};
```

Comment:
Hmmm so this is a good case of us to start asking the question, when to create an `indicator` or not?
- On the one hand, this is a valid concept within our PSSS system, and may prove to be re-used in other place
- On the other hand, this is just an aggregation on an existing indicator. Perhaps it would be best just to add support for aggregations in the report-server? (something we are planning on doing anyway)

I think personally I am leaning towards the later, however I'm still not entirely sure how best to structure the `aggregation` config in report-server.

If you're happy to go down that route, perhaps you, I, and @kael89 if he's interested could have a design discussion on the config?

Reply:
Yep I totally agree the latter option will definitely be more useful in the future, so I'm happy to go down that route. The only concern I have is time constraint given that we have to deliver phase 2 by end of this week, and I'm not too familiar with report-server to estimate how long it would take to add support for aggregation. If you think it's achievable then count me in 👍

Reply:
As discussed with Rohan on Slack. Ultimately, we want to achieve the aggregation pattern used in indicator package, which is to have separate aggregation per data element. But for now, we can keep it simple and just specify a single aggregation for all data elements as a default config, which is all required for this report and some upcoming STRIVE dashboards for solutions team. So for now the config could look like:

```
{
  fetch: {
    dataElements: [...],
    aggregation: 'SUM_PER_PERIOD'
  },
  transform: [...]
}
```

Reply:
Great call guys, I also agree! We could pull the aggregator config parsing out of `indicators` and into a common place (eg the `aggregator` package)